srcpkgs/lxc/template

```diff
@@ -26,13 +26,10 @@ make_dirs="
 pre_configure() {
 	# make sure the build uses the new docbook2x format,
 	# even though the command has the old "docbook2man" name
-	sed -e 's,xdocbook2man,xforcedocbook2x,g' -i configure.ac
-	./autogen.sh
+	vsed -e "s,sgml2man.full_path(),'defnot-docbook2man',g" -i meson.build
```

Comment:
Can you write a patch instead?

Reply:
Yeah should not be a problem. Still need to figure out what the heck is happening on musl though.

Reply:
Please also patch https://github.com/lxc/lxc/blob/master/meson.build#L119-L128 to
```
distrosysconfdir = join_paths(sysconfdir, 'default')
conf.set('LXC_DISTRO_SYSCONF', distrosysconfdir)
```

Reply:
```
distrosysconfdir = join_paths(sysconfdir, 'default')
conf.set('LXC_DISTRO_SYSCONF', distrosysconfdir)
```
Is the fix for musl

Reply:
Added a patch for docbook2x.

Regarding musl/distrosysconfdir: can't I also `mkdir -p /etc/default` in pre_configure? That worked alright for me. We would not be able to upstream much I else I think.

Reply:
I would prefer to not touch anything outside of builddir.

Reply:
thanks... that is how I was going to solve it but have not had the time ;)